softperipheral/doc/sQSPI/nrf54H20_porting_v0_1_0.rst

```diff
@@ -0,0 +1,231 @@
+.. _nrf54H20_porting_guide:
+
+nRF54H20 porting guide
```

Comment:
I find there is too much duplication between this section and the 54L15 one, but I will leave it to @b-gent to comment on it.

Reply:
I agree, there's a lot of duplication, but when I ran a diff on it, it showed few changes across the doc (additional tables and code-blocks), so if we were to create tabs for it I think it could be confusing. For now I'd leave it as is but I'll keep thinking of some other solution